.devcontainer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.163.1/containers/python-3/.devcontainer/base.Dockerfile
```

Comment:
This file and others in the .devcontainer folder allows you to run the project in a container (it's like a VM)